swap_meet/vendor.py

```diff
@@ -1,2 +1,56 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        self.inventory = inventory if inventory is not None else []
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
+
+    def get_by_id(self, id):
+        for item in self.inventory:
+            if item.id == id:
+                return item
+        return None
+
+    def swap_items(self, other_vendor, my_item, their_item):
+        if not (my_item in self.inventory and their_item in other_vendor.inventory):
+            return False
```

Comment:
Nice guard clause at the top of the method to quickly return if there's nothing to swap 👍